keps/sig-api-machinery/3716-admission-webhook-match-conditions/README.md

```diff
@@ -342,25 +338,23 @@ implementing this enhancement to ensure the enhancements have also solid foundat
 
 ##### Unit tests
 
-TBD - unit tests will be added as this feature is implemented.
-
 ##### Integration tests
 
 Test cases to add:
 
-- [ ] Feature gate enablement / disablement is a no-op when no `matchConditions` are set
-- [ ] Feature gate enablement / disablement works as expected when `matchConditions` are set
-- [ ] Single match condition:
-    - [ ] Request out of scope without `matchConditions`
-    - [ ] Request in scope without `matchConditions`, but not matching
-    - [ ] Request in scope without `matchConditions`, and also matching
-- [ ] Multiple match conditions, covering the same cases as the single-condition case
+- [X] Feature gate enablement / disablement is a no-op when no `matchConditions` are set
+- [] Feature gate enablement / disablement works as expected when `matchConditions` are set
```

Comment:
This test is a TODO I believe, we have one for testing the fields on the API object but not the functionality of the match conditions based on feature gate

Reply:
Good catch.  This might not matter now since we remove the feature gate and the feature is always on when we graduate to GA. Still, this edit is correct.  Maybe expand it with a comment that this got missed in Beta and that since we're now targeting GA we're about to delete the feature gate and don't need this?